tripy/tripy/types.py

```diff
@@ -33,15 +33,26 @@
     module=sys.modules[__name__],
     symbol="TensorLike",
     doc="""
-        Type annotation for a parameter that is either a Tripy :class:`Tensor` or a Python number that can be automatically converted into one.
+        A Tripy :class:`Tensor` or a Python number that can be automatically converted into one.
         """,
 )(Union["tripy.Tensor", numbers.Number])
 
+
+IntLike = export.public_api(
```

Comment:
When we allow FP shape tensors we will have to change the alias name again. Maybe use `NumberLike`?